raiden/utils/echo_node.py

```diff
@@ -65,7 +73,9 @@ def __init__(self, api: RaidenAPI, token_address: TokenAddress):
 
         self.num_seen_events = 0
         self.received_transfers: Queue[EventPaymentReceivedSuccess] = Queue()
-        self.stop_signal = None  # used to signal REMOVE_CALLBACK and stop echo_workers
+        self.stop_signal: Optional[
+            bool
+        ] = None  # used to signal REMOVE_CALLBACK and stop echo_workers
```

Comment:
super nitpick: I had to read this twice to realized that "this is used to.."

At first I thought it referred to the past due to "used to" and was wondering what the heck ...

Can you maybe change the comment to start with "this" or "it" ?